src/main/java/org/folio/inventory/dataimport/handlers/matching/loaders/HoldingLoader.java

```diff
@@ -73,6 +72,12 @@ protected String addCqlSubMatchCondition(DataImportEventPayload eventPayload) {
     return cqlSubMatch;
   }
 
+  private static boolean checkIfValueIsNullOrEmpty(String value) {
```

Comment:
@JavokhirAbdullayev , could you use another name for this method like "isNullOrEmpty"?